mobile/.metadata

```diff
@@ -1,10 +1,45 @@
 # This file tracks properties of this Flutter project.
 # Used by Flutter tool to assess capabilities and perform upgrades etc.
 #
-# This file should be version controlled and should not be manually edited.
```

Comment:
Was this file changed manually? This line specifically said it should not be manually edited

Reply:
No i didn't change manually

Reply:
Got it, thank you!